docs/framework/20sources/30implementation/access-control.md

```diff
@@ -1,7 +1,56 @@
 ### Role Based Access Control
-Role Based Access Control (RBAC) allows separate accounts to be 
-responsible for different actions. It allows sensitive operations 
-to be controlled by one multisig while other less sensitive 
-operations can be controlled by a different multisig. 
+[Role Based Access Control (RBAC)](https://en.wikipedia.org/wiki/Role-based_access_control) 
+allows different entites to be responsible for different configuration actions. With 
```

Comment:
I think that we can maybe add a sentence here that makes the key point of this section a bit more explicit, which is that a system managed by a credential with wide-ranging control potentially creates more security risks than one with narrowly-scoped privileges for different credentials and specific contexts.

Reply:
Done